packages/terra-application-menu-layout/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 3.5.0 - (June 12, 2019)
 ------------------
 ### Removed
+* Removed '-ms-overflow-style' style
```

Comment:
Looks like this entry ended up in the wrong section, we should move it back into Unreleased

Reply:
Fixed https://github.com/cerner/terra-framework/pull/728/commits/f6bc73acb8c9ff28ad329fa9f653d80f186a7e7f and https://github.com/cerner/terra-framework/pull/728/commits/5899a38012b4d48daae0e8d5690426fd3dbbc88b